python_modules/dagster/dagster/_core/definitions/multi_dimensional_partitions.py

```diff
@@ -273,7 +273,7 @@ def has_partition_key(
                 return False
         return True
 
-    @lru_cache(maxsize=5)
+    @lru_cache(maxsize=1)
```

Comment:
can you leave a comment explaining what the exact intent of the cache is ? IS it something like "hold results for repeated calls for the same `current_time`" after this change?